app.py

```diff
@@ -0,0 +1,24 @@
+from datetime import timedelta
+
+from union import Resources, ImageSpec
+from union.app import App, ScalingMetric
+
+zonos_image = ImageSpec(
+    base_image="pytorch/pytorch:2.6.0-cuda12.4-cudnn9-devel",
```

Comment:
Do not use a base image. The default base image + torch + cuda libraries from pypi, should work. (The cuda libraries are defined in your `uv.lock` file)

```suggestion
```